spark-dependencies/pom.xml

```diff
@@ -565,6 +565,15 @@
       </properties>
     </profile>
 
+    <profile>
```

Comment:
can you update to 2.1.2 and 2.2.1 above?

Reply:
Fixed

Reply:
@felixcheung , looks like spark-2.2.1 is not uploaded in http://d3kbcqa49mib13.cloudfront.net/spark-2.2.1.tgz which is used for zeppelin to download it for integration test. Can you help check what's missing in spark side ?

Reply:
Same issue for spark-2.1.2.  http://d3kbcqa49mib13.cloudfront.net/spark-2.1.2.tgz